File: Java/07fibonacci.java
public class 7fibonacci {

	public static void main(String[] args) {
		
		//7. - fibonacci
		int number = 10;
			
		
		int[] first = {0, 1};
		int sum = 0;
		
		for (int i = 1; i < number; i++) {
			
			System.out.println(sum);
			sum = first[0] + first[1];
			first[0] = first[1];
			first[1] = sum;
					
		}
		
		System.out.println("Last fibonacci sum: "+first[1]);
		
		
	}

}

File: Python/07fibonacci.py
#7. - fibonacci
def fibonacci(number):
   n1, n2 = 0, 1
   count = 0
   print("Last Fibonacci add of",number,":")
   while count < number:
         nth = n1 + n2

         n1 = n2
         n2 = nth
         count += 1
         if count == number:
            print(n1)
   
fibonacci(10)